Xamarin.Essentials/Platform/Platform.tizen.cs

```diff
@@ -22,6 +22,8 @@ public static partial class Platform
         static TizenOrientationSensor orientationSensor = null;
         static MapService mapService = null;
 
+        public static ElmSharp.Window MainWindow { get; set; }
```

Comment:
Like my first comment, I think this should be an Init method as it is consistent with the other platforms.

However, we _can_ (and probably should) have an `internal` property for the other APIs to use.

Reply:
public method is ok ?